plugin.py

```diff
@@ -271,7 +264,7 @@ def pulls(self, irc, msg, args, slug):
 
         try:
             results = sum([
-                list(self.yield_github_pulls(username, repo)) for repo in repos
+                list(self.yield_github_pulls(username, r)) for r in repos
```

Comment:
Yeah, it is that ``repo`` here is redefining a variable that is used in a preceding line.